### ISSUE TITLE
Add profiles

### ISSUE THREAD

**Issue by paltman** [pinax/patch-game#8]

**Comment by jtauber** [comment]
I think this can definitely come later, perhaps just to motivate more app development.

**Comment by jtauber** [comment]
Although I guess we could have a page for each user that shows points, level, and badges. Is this all you meant @paltman or with user-edited information too?

**Comment by paltman** [comment]
yes, i meant just entirely a place to hang stuff on.  a user detail page.

**Comment by jtauber** [comment]
cool, then we'll need it sooner just to show a user's points and level and eventually badges (and the leaderboard can link to it for any user)
